aeron-driver/src/main/c/aeron_flow_control.h

```diff
@@ -27,9 +27,8 @@
 
 #define AERON_MIN_FLOW_CONTROL_RECEIVERS_COUNTER_NAME ("fc-receivers")
 
-#define AERON_MAX_FLOW_CONTROL_RETRANSMIT_RECEIVER_WINDOW_MULTIPLE (4)
+#define AERON_MULTICAST_FLOW_CONTROL_RETRANSMIT_RECEIVER_WINDOW_MULTIPLE (4)
 #define AERON_UNICAST_FLOW_CONTROL_RETRANSMIT_RECEIVER_WINDOW_MULTIPLE (16)
```

Comment:
Should be declared as unsigned, i.e. add `UINT32_C` before the value (e.g. `UINT32_C(4)`).